pkg/services/notifications/mailer.go

```diff
@@ -23,6 +24,8 @@ func (ns *NotificationService) send(msg *Message) (int, error) {
 		return 0, err
 	}
 
+	var num int
+	var werr error
```

Comment:
error variables should be named `err`. You can just avoid declaring this variable and overwrite `err` instead.

Reply:
ok